arangod/Cluster/ClusterRepairDistributeShardsLike.cpp

```diff
@@ -412,7 +412,7 @@ ResultT<std::list<RepairOperation>> DistributeShardsLikeRepairer::fixShard(
       createFixServerOrderOperation(collection, proto, shardId, protoShardId);
 
   if (maybeFixServerOrderOperationResult.fail()) {
-    return std::move(maybeFixServerOrderOperationResult);
+    return std::move(maybeFixServerOrderOperationResult.stealResult());
```

Comment:
Superfluous `std::move` - `stealResult` already returns an rvalue reference (obviously applies to all the other stealResult calls as well).